MesaModel/model/Pupil.py

```diff
@@ -98,6 +100,9 @@ def redStateCange(self):
             self.time_in_green_state = 0
             return 1
 
+        # If more neighbours are disruptive than a randomly generated
+        # number and the pupil is more disruptive than the average
+        # pupil, pupil will become passive
```

Comment:
Worth mentioning that we're comparing with the mean disruptiveTend (more specific than 'average')?